config/example.yaml

```diff
@@ -8,7 +8,14 @@ receivers:
   hostmetrics:
     collection_interval: 1m
     scrapers:
+      # Theses scrapers work on all operating systems
       load:
+      filesystem:
+      memory:
+      network:
+      # These scrapers do not support MacOS please comment them out to reduce errors in collector logs
```

Comment:
Rather than "please comment" maybe "Remove these scrapers for MacOS collectors as they are not supported and will generate errors in the logs"

Reply:
I like that.

Reply:
Also I believe `macOS` is the official spelling